packit/sync.py

```diff
@@ -0,0 +1,47 @@
+import glob
+import os
+import shutil
+import logging
+
+from packit.exceptions import PackitException
+from packit.config import PackageConfig, SyncFilesItem, SyncFilesConfig
+
+logger = logging.getLogger(__name__)
+
+
+def get_wildcard_resolved_sync_files(pc: PackageConfig) -> None:
+    logger.debug("Packit synced files %s", pc.synced_files.files_to_sync)
+    files_to_sync = []
+    for sync in pc.synced_files.files_to_sync:
+        if isinstance(sync, str):
+            files_to_sync.append(SyncFilesItem(src=sync, dest=sync))
+            continue
+        src = glob.glob(sync.src)
+        if src:
+            for s in src:
+                files_to_sync.append(SyncFilesItem(src=s, dest=sync.dest))
+    logger.debug(f"Resolved synced file {files_to_sync}")
+    pc.synced_files = SyncFilesConfig(files_to_sync=files_to_sync)
+
+
+def sync_files(pc: PackageConfig, src_working_dir: str, dest_working_dir: str) -> None:
```

Comment:
I suggest to also expect packit Config  parameter -> to be able to read `debug` option, or in future `dry run` and skip real copying

Reply:
This can be done later on. But good point. https://github.com/packit-service/packit/issues/232